includes/batch.inc

```diff
@@ -537,3 +585,105 @@ function islandora_paged_content_create_page_from_pdf_batch_operation(array $pag
 
   }
 }
+
+/**
+ * Batch definition for consolidating missing OCR.
+ */
+function islandora_paged_content_consolidate_missing_ocr_batch() {
+  return array(
+    'title' => t('Batch Consolidate Missing OCR'),
+    'init_message' => t('Preparing to consolidate missing OCR ...'),
+    'progress_message' => t('Time elapsed: @elapsed <br/>Estimated time remaining: @estimate.'),
+    'error_message' => t('An error has occured.'),
+    'file' => drupal_get_path('module', 'islandora_paged_content') . '/includes/batch.inc',
+    'operations' => array(
+      array('islandora_paged_content_consolidate_missing_ocr_batch_operation', array()),
+    ),
+  );
+}
+
+/**
+ * Batch operation for consolidating missing OCR.
+ */
+function islandora_paged_content_consolidate_missing_ocr_batch_operation(&$context) {
+  module_load_include('inc', 'islandora_paged_content', 'includes/utilities');
+  $connection = islandora_get_tuque_connection();
+  $repository = $connection->repository;
+  $sandbox = &$context['sandbox'];
+  if (empty($sandbox)) {
+    $context['finished'] = 0;
+    $sandbox['objects_completed'] = 0;
+    $sandbox['objects'] = islandora_paged_content_get_missing_ocr_paged_content_objects();
+    $sandbox['total'] = count($sandbox['objects']);
```

Comment:
Should probably drop out if we find nothing... otherwise:
```
No missing OCR found on paged content objects.                                                                                                     [ok]
Division by zero batch.inc:659                                                                                                                     [warning]
No missing OCR found on paged content objects.                                                                                                     [ok]
Division by zero batch.inc:659                                                                                                                     [warning]
[ ... repeats until interrupted ...]
```